SPECS/gnutls/gnutls.spec

```diff
@@ -1,13 +1,17 @@
 Summary:        The GnuTLS Transport Layer Security Library
 Name:           gnutls
 Version:        3.8.3
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        GPLv3+ AND LGPLv2.1+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Libraries
 URL:            https://www.gnutls.org
 Source0:        https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/%{name}-%{version}.tar.xz
+# Patch taken from 3.8.4 release
+Patch:          CVE-2024-28834.patch
```

Comment:
Why not Patch1, Patch2 ?

Reply:
I recall reading that un-numbered patches were generally preferred now upstream, but I can't for the life of me find it again... easy to just add the explicit numbers back (but it does work without).

Reply:
Fixed